packages/ui/src/components/va-pagination/VaPagination.demo.vue

```diff
@@ -1,216 +1,376 @@
               />
               <va-pagination
+                v-model="activePage"
                 :pages="10"
                 :visible-pages="3"
-                boundary-icon-left='volume_off'
-                boundary-icon-right='volume_up'
-                direction-icon-right='volume_down'
-                direction-icon-left='volume_mute'
+                boundary-icon-left="volume_off"
+                boundary-icon-right="volume_up"
+                direction-icon-left="volume_mute"
+                direction-icon-right="volume_down"
+              />
+            </VbCard>
+          </td>
+        </tr>
+        <tr>
+          <td>Input</td>
+          <td>
+            <VbCard>
+              <va-pagination
                 v-model="activePage"
+                :pages="20"
+                input
               />
             </VbCard>
           </td>
         </tr>
-      </table>
-    </VbCard>
-    <VbCard
-      title="With input"
-    >
-      <va-pagination
-        :pages="20"
-        v-model="activePage"
-        input
-      />
-    </VbCard>
-    <VbCard
-      title="With boundary numbers"
-    >
-      <va-pagination
-        :pages="20"
-        :visible-pages="5"
-        v-model="activePage"
-        boundary-numbers
-      />
-      <va-pagination
-        :pages="20"
-        :visible-pages="7"
-        v-model="activePage"
-        boundary-numbers
-      />
-    </VbCard>
-    <VbCard
-      title="Flat"
-    >
-      <va-pagination
-        :pages="20"
-        :visible-pages="5"
-        v-model="activePage"
-        flat
-      />
-    </VbCard>
-    <VbCard
-      title="Flat with input"
-    >
-      <va-pagination
-        :pages="20"
-        v-model="activePage"
-        flat
-        :visible-pages="5"
-        input
-      />
-    </VbCard>
-    <VbCard
-      title="Hide on single page"
-    >
-      <va-pagination
-        :pages="1"
-        hide-on-single-page
-      />
-    </VbCard>
-    <VbCard
-      title="Stateful"
-    >
-      <va-pagination
-        :pages="10"
-        stateful
-        :visible-pages="5"
-      />
-    </VbCard>
-    <VbCard
-      title="Use context"
-    >
-      <va-config :components="{
+        <tr>
+          <td>Boundary Icons</td>
+          <td>
+            <VbCard>
+              <va-pagination
+                v-model="activePage"
+                :pages="20"
+                :visible-pages="5"
+                boundary-numbers
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="20"
+                :visible-pages="7"
+                boundary-numbers
+              />
+            </VbCard>
+          </td>
+        </tr>
+        <tr>
+          <td>Plain with input</td>
+          <td>
+            <VbCard>
+              <va-pagination
+                v-model="activePage"
+                :pages="20"
+                :visible-pages="5"
+                input
+                plain
```

Comment:
Plain prop is at very weird spot right now. It works only with input (and it actually forces input), maybe we can remove it and somehow make visuals rely on preset prop?

Here's what it does for `default` pagination demo. Doesn't look intended.
<img width="347" alt="image" src="https://user-images.githubusercontent.com/5394573/189236616-0af877f9-5db4-4945-b332-7f4de5cb22c4.png">